joern-cli/frontends/rubysrc2cpg/src/test/scala/io/joern/rubysrc2cpg/passes/ast/MethodTwoTests.scala

```diff
@@ -0,0 +1,104 @@
+package io.joern.rubysrc2cpg.passes.ast
+
+import io.joern.rubysrc2cpg.testfixtures.RubyCode2CpgFixture
+import io.shiftleft.codepropertygraph.generated.{EvaluationStrategies, NodeTypes}
+import io.shiftleft.semanticcpg.language.*
+import io.shiftleft.semanticcpg.language.types.structure.NamespaceTraversal
+
+class MethodTwoTests extends RubyCode2CpgFixture {
+
+  "Method test with define_method" should {
+    val cpg = code("""
+        |define_method(:foo) do |a, b|
+        |  return ""
+        |end
+        |""".stripMargin)
+
+    // TODO: This test cases needs to be fixed.
+    "should contain exactly one method node with correct fields" ignore {
+      inside(cpg.method.name("foo").l) { case List(x) =>
+        x.name shouldBe "foo"
+        x.isExternal shouldBe false
+        x.fullName shouldBe "Test0.rb::program:foo"
+        x.code should startWith("def foo(a, b)")
+        x.isExternal shouldBe false
+        x.order shouldBe 1
+        // TODO: At present it is returning absolute path
+        //        x.filename shouldBe "Test0.rb"
```

Comment:
Here again we can check `endsWith`

Reply:
@DavidBakerEffendi  what is expected full file path or relative file path?